services/teams/resolvers.ts

```diff
@@ -0,0 +1,61 @@
+import logger from './utilities/logger'
+import { Pool } from 'pg'
+import { get, isEmpty } from 'lodash'
+import { hash, compare } from 'bcrypt'
+import { sign, verify } from 'jsonwebtoken'
+
+export const jwtAlgorithm = 'HS512'
+
+const pool = new Pool({
+    host: 'postgres',
+    user: 'postgres',
+    password: 'changeme',
+    port: 5432
+})
+
+interface Team {
+  id: number
+  name: String
+  description: String
+}
+interface Team_Challenges {
+    id: number
+    teamId: number // FK
+    challengeId: number // FK
+}
+interface Team_Members {
+    id: number
+    teamId: number // FK
+    userId: number // FK
+}
+interface Challenge {
+    id: number
+    description: String
+    createdByUserId: number // FK, Defines whether it's a team project (created by a user of the project) or sponsered challenge (created by a "user" of the sponser)
+}
+
+export default {
+    Query: {
+        register: async (parent, { id }) => {
```

Comment:
Not seeing any query here that would require this over Hasura pg schemas.

Reply:
Removed team service.